posydon/popsyn/binarypopulation.py

```diff
@@ -510,8 +510,10 @@ def combine_saved_files(self, absolute_filepath, file_names, **kwargs):
         
         
         with pd.HDFStore(absolute_filepath, mode=mode, complevel=complevel, complib=complib) as store:
-            simulated_mass = 0
-            number_of_systems = 0
+            number_of_systems=0
```

Comment:
I think you shouldn't remove the initialization of the `simulated_mass`, here. It will be undefined in case your try later fails, while it will always being accessed in `data={'simulated_mass': simulated_mass, ...`.